### DISCUSSION TITLE
Custom wrapper can't seem to work

### DISCUSSION THREAD

**Discussion by krausestudio** [darkroomengineering/lenis#141] [General]
First and foremost—love your library!

Though after the update I can't seem to define the wrapper. I would like for my <main> to be the wrapper(meaning that the lenis classes will be added to <main>)

Can you assist me on the correct implementation of this?

Current:
`const lenis = new Lenis({
  wrapper: document.main, // what to add here
  duration: 1.2,
  mouseMultiplier: 1,
  smoothTouch: true,
  infinite: false
});
lenis.on("scroll", (e) => {
  console.log(e);
});
function raf(time) {
  lenis.raf(time);
  requestAnimationFrame(raf);
}
requestAnimationFrame(raf);
`

**Comment by clementroche** [comment]
You need to provide content element as well

for example: 
wrapper: document.body
content : document.main
